.github/workflows/build.yml

```diff
@@ -112,7 +112,6 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.6"
```

Comment:
If you are removing support for Python 3.6, you should also remove [the 3.6 line in setup.py](https://github.com/cisagov/pe-reports/blob/develop/setup.py#L76) and add a comment there explaining why 3.6 cannot be supported.

Reply:
You will also need to update [this line](https://github.com/cisagov/pe-reports/blob/develop/setup.py#L83) as well.

Reply:
commit efec55caa3fa07b6f9f747208fb131052059d698

Reply:
Since this is a temporary issue until `skeleton-python-library` is updated, you should comment out the lines related to 3.6 (as opposed to removing them) and create an issue in this repo as a reminder to change this back later.  After you create the issue, update the comment to reference it.

Reply:
created the issue and added it as comment. commit a371a5455ed5aed054e392f1ce0f719c206887c0

Reply:
Thanks @edujosemena - I added a [suggestion](https://github.com/cisagov/pe-reports/pull/259#discussion_r1059432227) to improve the language a bit.